tests/Jellyfin.Server.Implementations.Tests/Library/PathExtensionsTests.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Emby.Server.Implementations.Library;
+using Xunit;
+
+namespace Jellyfin.Server.Implementations.Tests.Library
+{
+    public class PathExtensionsTests
+    {
+        [Theory]
+        [InlineData("Superman: Red Son [imdbid=tt10985510]", "imdbid", "tt10985510")]
+        [InlineData("Superman: Red Son - tt10985510", "imdbid", "tt10985510")]
+        [InlineData("Superman: Red Son", "imdbid", null)]
+        public void GetAttributeValue_ValidArgs_Correct(string input, string attribute, string? result)
```

Comment:
Thoughts on this conventions? I think it's nice to be a bit more descriptive than just "`result`" and it makes it clear which parameters are inputs, and which are expected values.

```suggestion
        public void GetAttributeValue_ValidArgs_Correct(string input, string attribute, string? expectedResult)
```